opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -420,7 +440,11 @@ def __new__(
         if trace_state is None:
             trace_state = DEFAULT_TRACE_STATE
 
-        is_valid = trace_id != INVALID_TRACE_ID and span_id != INVALID_SPAN_ID
+        is_valid = (
+            trace_id != INVALID_TRACE_ID
+            and span_id != INVALID_SPAN_ID
```

Comment:
Instead, we should just put the code of `_validate_trace_id` here instead of having a single-use function. Keep in mind that the only thing that `_validate_trace_id` does is `trace_id < 2 ** 128`.